src/core/public/injected_metadata/injected_metadata_service.ts

```diff
@@ -48,6 +50,9 @@ export interface InjectedMetadataParams {
     csp: {
       warnLegacyBrowsers: boolean;
     };
+    externalUrl: {
+      policy: ExternalUrlConfig['policy'];
```

Comment:
Nit: could be simplified

```suggestion
      policy: IExternalUrlPolicy[];
```